src/fides/api/main.py

```diff
@@ -315,22 +307,22 @@ def read_other_paths(request: Request) -> Response:
 
     # If any of those worked, serve the file.
     if ui_file and ui_file.is_file():
-        log.debug(
+        logger.debug(
             f"catchall request path '{path}' matched static admin UI file: {ui_file}"
         )
```

Comment:
Should we reformat all of these f string instances while we're here to postpone the formatting of this error message like you're doing in some of the other logs:  At one point, we'd gotten rid of all of these f strings in the logs on the ops side.

```
logger.info("Finding connection configuration with key '{}'", connection_key)
```

Reply:
yeah, i saw this and sort of deferred. but this nudge has given me the motivation to do it! i'll go for it.

Reply:
On the ops side we'd done this https://github.com/ethyca/fidesops/issues/837#issuecomment-1178934542 to find all instances and correct - you could probably flip a rule in pylint to catch all these

Reply:
not sure that's going to work (as a way to identify the instances) for us now though since we're not using the standard `logging` library? i don't see that line in the `pyproject.toml` and my pylint doesn't complain about anything...maybe i'm missing something though?

that being said, i think i can probably identify the instances with a regex

Reply:
sounds good you're right I don't see that line here in fides